### PR TITLE
Remove licence from dependency

### PR DESCRIPTION
**Pull request by cvictory**
(BEA licensed) "Java Concurrency in Practice" book annotations
(net.jcip:jcip-annotations:1.0 - http://jcip.net/)

For example, in dubbo-registry/dubbo-registry-sofa:

[INFO] +- org.jboss.resteasy:resteasy-jaxrs:jar:3.0.19.Final:test
[INFO] |  +- org.jboss.spec.javax.ws.rs:jboss-jaxrs-api_2.0_spec:jar:1.0.0.Final:test
[INFO] |  +- org.jboss.spec.javax.annotation:jboss-annotations-api_1.2_spec:jar:1.0.0.Final:test
[INFO] |  +- javax.activation:activation:jar:1.1.1:test
[INFO] |  +- org.apache.httpcomponents:httpclient:jar:4.5.3:test
[INFO] |  |  +- org.apache.httpcomponents:httpcore:jar:4.4.6:test
[INFO] |  |  \- commons-codec:commons-codec:jar:1.9:test
[INFO] |  +- commons-io:commons-io:jar:2.1:test
[INFO] |  +- net.jcip:jcip-annotations:jar:1.0:test
[INFO] |  \- org.jboss.logging:jboss-logging:jar:3.1.4.GA:test

     (GNU LESSER GENERAL PUBLIC LICENSE, Version 2.1) SpotBugs
Annotations (com.github.spotbugs:spotbugs-annotations:3.1.10 -
https://spotbugs.github.io/)

For example, in dubbo-configcenter-etcd:

[INFO] +- io.etcd:jetcd-launcher:jar:0.3.0:test
[INFO] |  +- com.github.spotbugs:spotbugs-annotations:jar:3.1.10:test
[INFO] |  \- org.slf4j:slf4j-api:jar:1.7.25:compile

### CODE REVIEW

**Comment by cvictory** [comment]
I have run licenseCheck.sh, and the dependency is not exist.

File: dubbo-dependencies-bom/pom.xml [hunk 1]
```diff
@@ -363,6 +363,12 @@
                 <groupId>org.jboss.resteasy</groupId>
                 <artifactId>resteasy-jaxrs</artifactId>
                 <version>${resteasy_version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>net.jcip</groupId>
+                        <artifactId>jcip-annotations</artifactId>
```
Comment (ralf0131) [inline dubbo-dependencies-bom/pom.xml]: This is no longer needed to be excluded. See discussion on mailing list.

**Comment by ralf0131** [comment]
Please remove the license-list.txt-e file.

File: dubbo-dependencies-bom/pom.xml (hunk 1 above)
Comment (cvictory) [reply dubbo-dependencies-bom/pom.xml]: I have revert it.

**Pull request merged**